src/data/finset/lattice.lean

```diff
@@ -77,6 +77,13 @@ lemma comp_sup_eq_sup_comp_of_is_total [is_total α (≤)] {γ : Type} [semilatt
   (g : α → γ) (mono_g : monotone g) (bot : g ⊥ = ⊥) : g (s.sup f) = s.sup (g ∘ f) :=
 comp_sup_eq_sup_comp g mono_g.map_sup bot
 
+/-- The first component of a sup in a subtype is the sup if first components. -/
```

Comment:
In addition to the typo if -> of, I think referring to the "first component" when the lemma is stated in terms of coercion is confusing.

Reply:
You're right. The docstring didn't get updated after I fixed the errors caused by #3204.